bookdown/_output.yml

```diff
@@ -23,8 +26,12 @@ bookdown::gitbook:
        <li><a href="./">Made with <i class="fas fa-heart"></i> for open-source </a></li>
 
 bookdown::pdf_book:
+  author: ""
```

Comment:
Why is this empty now?

Reply:
Otherwise we get a single entry on the first page with just the author name. This is how pandoc does it.
I'd prefer to start the book with the titlepage.